BREAKING_CHANGES.md

```diff
@@ -46,33 +55,91 @@ this field and its corresponding behavior has been removed.
 
 ## Stencil v3.0.0
 
-* [General](#general)
-  * [New Configuration Defaults](#new-configuration-defaults)
-    * [SourceMaps](#sourcemaps)
-    * [`dist-custom-elements` Type Declarations](#dist-custom-elements-type-declarations)
-  * [Legacy Browser Support Fields Deprecated](#legacy-browser-support-fields-deprecated)
-    * [`dynamicImportShim`](#dynamicimportshim)
-    * [`cssVarsShim`](#cssvarsshim)
-    * [`shadowDomShim`](#shadowdomshim)
-    * [`safari10`](#safari10)
-  * [Deprecated `assetsDir` Removed from `@Component()` decorator](#deprecated-assetsdir-removed-from-component-decorator)
-  * [Drop Node 12 Support](#drop-node-12-support)
-  * [Strongly Typed Inputs](#strongly-typed-inputs)
-  * [Narrowed Typing for `autocapitalize` Attribute](#narrowed-typing-for-autocapitalize-attribute)
-  * [Custom Types for Props and Events are now Exported from `components.d.ts`](#custom-types-for-props-and-events-are-now-exported-from-componentsdts)
-  * [Composition Event Handlers Renamed](#composition-event-handlers-renamed)
-* [Output Targets](#output-targets)
-  * [`dist-custom-elements` Output Target](#dist-custom-elements-output-target)
-    * [Add `customElementsExportBehavior` to Control Export Behavior](#add-customelementsexportbehavior-to-control-export-behavior)
-    * [Move `autoDefineCustomElements` Configuration](#move-autodefinecustomelements-configuration)
-    * [Remove `inlineDynamicImports` Configuration](#remove-inlinedynamicimports-configuration)
-  * [`dist-custom-elements-bundle` Output Target](#dist-custom-elements-bundle-output-target)
-* [Legacy Angular Output Target](#legacy-angular-output-target)
-* [Stencil APIs](#stencil-apis)
-  * [Flag Parsing, `parseFlags()`](#flag-parsing-parseflags)
-  * [Destroy Callback, `addDestroy()`, `removeDestroy()`](#destroy-callback-adddestroy-removedestroy)
-* [End-to-End Testing](#end-to-end-testing)
-  * [Puppeteer v10+ Required](#puppeteer-v10-required)
+- [Breaking Changes](#breaking-changes)
```

Comment:
Is this a rebase problem or am I missing something? Wouldn't think anything in the v3 section should change